sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventProcessorBuilder.java

```diff
@@ -44,19 +62,141 @@
 
     private final ClientLogger logger = new ClientLogger(EventProcessorBuilder.class);
 
-    private Supplier<PartitionProcessor> partitionProcessorFactory;
+    private final EventHubClientBuilder eventHubClientBuilder;
     private String consumerGroup;
-    private PartitionManager partitionManager;
-    private EventHubAsyncClient eventHubAsyncClient;
+    private EventProcessorStore eventProcessorStore;
+    private Function<PartitionEvent, Mono<Void>> processEvent;
+    private Function<EventProcessingErrorContext, Mono<Void>> processError;
+    private Function<InitializationContext, Mono<Void>> initializePartition;
+    private Function<CloseContext, Mono<Void>> closePartition;
+
+    public EventProcessorBuilder() {
```

Comment:
missing javadoc.
Should we mention that this `creates a new instance  of EventhubClientBuilder with the default transport type AMQP`